This patch sets the default value of the internal variable disable_job_logging
to TRUE.  This avoids a memory leak if the directory that Upstart is trying to
write log files to (/var/log/upstart) doesn't exist.
Since we do not want Upstart to be doing logging anyway there is no code
added to provide a command line flag to override this change.
--- a/init/job_process.c
+++ b/init/job_process.c
@@ -120,7 +120,7 @@ static void job_process_remap_fd        (int *fd, int reserved_fd, int error_fd)
  * If TRUE, do not log any job output.
  *
  **/
-int disable_job_logging = 0;
+int disable_job_logging = TRUE;
 
 /**
  * no_inherit_env:
